DependencyInjection/DunglasActionExtension.php

```diff
@@ -34,7 +34,8 @@ public function load(array $configs, ContainerBuilder $container)
 
         $directoryList = [];
         foreach ($config['directories'] as $pattern) {
-            list($classes, $directories) = $this->getClasses($kernelRootDir.DIRECTORY_SEPARATOR.$pattern);
+            $directory = (DIRECTORY_SEPARATOR !== substr($pattern, 0, 1)) ? $kernelRootDir.DIRECTORY_SEPARATOR.$pattern : $pattern;
```

Comment:
hm so eventually we should check both (you may give a hardcoded path `/path/to`. As for `%kernel.root_dir%` it may differs as it depends to the platform (may be wrong there didn't check)